guides/source/active_record_basics.md

```diff
@@ -606,6 +607,16 @@ irb> user.save!
 ActiveRecord::RecordInvalid: Validation failed: Name can't be blank
 ```
 
+The `create` method in the example returns an ActiveRecord object, regardless of
```

Comment:
I think we can be a bit stricter here, `create` always returns an ActiveRecord object:
```suggestion
The `create` method always returns an ActiveRecord object, regardless of
```

Reply:
@p8 
sure, done.